column/__init__.py

```diff
@@ -9,7 +10,7 @@
 
 import ConfigParser
 
-DEFAULT_CONF_FILE = '/etc/column/column.conf'
+DEFAULT_CONF_FILE = os.path.join('/etc/column', 'column.conf')
```

Comment:
So when running in a virtualenv like tox, using an absolute path can cause problems.